css/quiz.css

```diff
@@ -0,0 +1,15 @@
+.carCard {
+  border: 1px dotted black; }
+
+nav {
+  margin-left: 730px; }
+
+.selectedCard {
+  border: 2px solid;
+  background-color: lightgreen; }
+
+  .spacer {
+	margin: 5px;
+}
+
+/*# sourceMappingURL=quiz.css.map */
```

Comment:
Remember, you don't need to push up this file. It will be generated when Grunt is run.